rotkehlchen/db/utils.py

```diff
@@ -79,29 +79,23 @@ def form_query_to_filter_timestamps(
         timestamp_attribute: str,
         from_ts: Optional[Timestamp],
         to_ts: Optional[Timestamp],
-) -> Tuple[str, Union[Tuple, Tuple[Timestamp], Tuple[Timestamp, Timestamp]]]:
-    """Formulates the query string and its bindings to filter for timestamps"""
-    bindings: Union[Tuple, Tuple[Timestamp], Tuple[Timestamp, Timestamp]] = ()
+) -> str:
+    """Formulates the query string for filterling by timestamps"""
     got_from_ts = from_ts is not None
     got_to_ts = to_ts is not None
-    if (got_from_ts or got_to_ts):
-        if 'WHERE' not in query:
-            query += 'WHERE '
-        else:
-            query += 'AND '
+    if got_from_ts or got_to_ts:
+        query += 'WHERE' if 'WHERE' not in query else 'AND '
 
+    filters = []
     if got_from_ts:
-        query += f'{timestamp_attribute} >= ? '
-        bindings = (from_ts,)
-        if got_to_ts:
-            query += f'AND {timestamp_attribute} <= ? '
-            bindings = (from_ts, to_ts)
-    elif got_to_ts:
-        query += f'AND {timestamp_attribute} <= ? '
```

Comment:
"the problem" was here. It seems Bitstamp is the only place where this function has been called passing only `to_ts`. I didn't spotted it because I mock the db in the unit tests, and I reckon I reused already existing used_query_ranges during manual testing...